src/Config.ts

```diff
@@ -12,7 +12,6 @@ export interface ProjectRoot {
 export interface Config {
 	projectRoot?: string;
 	roots?: ProjectRoot[];
-	namespaceSeparator?: string;
```

Comment:
Any specific reason why you removed this again? I think it's a good feature, regardless of its previous necessity for bash support.

Reply:
Just going for minimal changes, can bring it back